.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,76 @@
+name: Continuous integration
+on:
+  - push
+jobs:
+  build:
+    name: "JDK ${{ matrix.java }} on ${{ matrix.os }}"
+    strategy:
+      matrix:
+        include:
+          - os: macos-latest
+            java: 8
+          - os: ubuntu-latest
+            java: 8
+          - os: ubuntu-latest
+            java: 11
+      fail-fast: false
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Check out NullAway sources
+        uses: actions/checkout@v2
+      - name: Cache Gradle caches
+        uses: actions/cache@v1
+        with:
+          path: ~/.gradle/caches
+          key: ${{ runner.os }}-gradle-caches-${{ hashFiles('**/*.gradle') }}
+          restore-keys: ${{ runner.os }}-gradle-caches-
+      - name: Cache Gradle wrapper
+        uses: actions/cache@v1
+        with:
+          path: ~/.gradle/wrapper
+          key: ${{ runner.os }}-gradlew-wrapper-${{ hashFiles('gradle/wrapper/gradle-wrapper.properties') }}
+          restore-keys: ${{ runner.os }}-gradlew-wrapper-
+      - name: 'Set up JDK ${{ matrix.java }}'
+        uses: actions/setup-java@v1
+        with:
+          java-version: ${{ matrix.java }}
+      - name: Build and test using Gradle and Java 8
+        run: ./gradlew verGJF build
+        if: matrix.java == '8'
+      - name: Build and test using Gradle and Java 11
+        run: ./gradlew :nullaway:test
+        if: matrix.java == '11'
+      - name: Report jacoco coverage
+        run: ./gradlew jacocoTestReport coveralls
```

Comment:
Ah, let me look into that a bit. Might be ok, but we are not in a rush.

Reply:
Since the author of [kt3k/coveralls-gradle-plugin](https://github.com/kt3k/coveralls-gradle-plugin) is looking for a new maintainer (see https://github.com/kt3k/coveralls-gradle-plugin/issues/107) and [nbaztec/coveralls-jacoco-gradle-plugin](https://github.com/nbaztec/coveralls-jacoco-gradle-plugin) supports GitHub Actions out-of-the-box, it might be worth migrating to the latter.

https://github.com/nbaztec/coveralls-jacoco-gradle-plugin#github-actions